yb-voyager/cmd/importData.go

```diff
@@ -109,6 +110,24 @@ func importDataCommandFn(cmd *cobra.Command, args []string) {
 	checkExportDataDoneFlag()
 	sourceDBType = GetSourceDBTypeFromMSR()
 	sqlname.SourceDBType = sourceDBType
+
+	if tconf.TargetDBType == YUGABYTEDB {
+		tconf.Schema = strings.ToLower(tconf.Schema)
+	} else if tconf.TargetDBType == ORACLE && !utils.IsQuotedString(tconf.Schema) {
+		tconf.Schema = strings.ToUpper(tconf.Schema)
+	}
+	tdb = tgtdb.NewTargetDB(&tconf)
+	err := tdb.Init()
+	if err != nil {
+		utils.ErrExit("Failed to initialize the target DB: %s", err)
+	}
+	defer tdb.Finalize()
```

Comment:
here calling `defer tdb.Finalize()`, will run this `Finalize()` even after the `export data from target` is completed ? as we have this at the end of this function. I think we should call this in `importData()` itself?
```
if changeStreamingIsEnabled(importType) {
		startExportDataFromTargetIfRequired()
	}
```

Reply:
did not add to `importdata()` because it is called by import data file command as well. 
added it in `importDataCommandFn` after calling `importData`